.github/workflows/restore-sanitized-db.yml

```diff
@@ -0,0 +1,102 @@
+name: Restore Sanitized DB
+
+concurrency: stage
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '50 1 * * MON' # At 01:50 UTC every Monday
+
+jobs:
+  sanitize-db:
+    runs-on: ubuntu-latest
+    environment: Production
+
+    env:
+      MYSQL_ROOT_PASSWORD: root
+
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - "3306:3306"
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=10s --health-retries=10
+
+    steps:
+      - name: Download DB backup
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: backup-db.yml
+          workflow_conclusion: success
+          check_artifacts: true
+
+      - name: Unzip artifact & load DB
+        env:
+          BACKUP_PASSWORD: ${{ secrets.BACKUP_PASSWORD }}
+        run: |
+          cd `ls -t | head -n 1`
+          unzip -P $BACKUP_PASSWORD `ls -t | head -n 1`
+          cat `ls -t *.sql | head -n 1` | /usr/bin/mysql -h 127.0.0.1 -u root -p$MYSQL_ROOT_PASSWORD
+
+      - name: Remove PII
+        run: |
+          cat <<EOF | /usr/bin/mysql -h 127.0.0.1 -u root -p$MYSQL_ROOT_PASSWORD --database orbitar_db
+          delete from invites;
```

Comment:
I'll add a separate sql file to do that (it will be a bit more involved than that), but this works as a first version.

Reply:
Sounds good, why is it more involved?

Reply:
One example: ideally we don't want to remove invites, but rather randomize their hashes (since removing invites breaks some features).

Also, we need to clear `sessions` table as well. 

In the future, a standalone SQL file makes more sense from the organizational standpoint.